dev-packages/e2e-tests/test-applications/nextjs-app-dir/tests/route-handlers.test.ts

```diff
@@ -60,6 +60,9 @@ test('Should record exceptions and transactions for faulty route handlers', asyn
 
   expect(routehandlerError.exception?.values?.[0].value).toBe('route-handler-error');
 
+  expect(routehandlerError.request?.method).toBe('PUT');
+  expect(routehandlerError.request?.url).toContain('/route-handlers/[param]/error');
```

Comment:
I think what we really need here is the unparameterized url. Which is a major pain in the butt to grab in Next.js because it turns static routes dynamic 🤔 Let me sit on this a bit.